test/ExternalBribes.t.sol

```diff
@@ -33,13 +34,24 @@ contract ExternalBribesTest is BaseTest {
         // deployVoter()
         gaugeFactory = new GaugeFactory();
         bribeFactory = new BribeFactory();
-        voter = new Voter(address(escrow), address(factory), address(gaugeFactory), address(bribeFactory));
+        voter = new Voter(
+            address(escrow),
+            address(factory),
+            address(gaugeFactory),
+            address(bribeFactory)
+        );
 
         escrow.setVoter(address(voter));
 
+        pairfactory.setVoter(address(voter));
```

Comment:
@duncan4123  are you supposed to test something with this?

Reply:
if voter is not set then we cant set externalBribe on pair as voter is required to be the caller